ee/ucp/kubernetes/storage/use-azure-files.md

```diff
@@ -238,6 +239,48 @@ spec:
 EOF
 ```
 
+### Troubleshooting
+
+When creating Persistent Volume Claims, the volume may constantly stay in a
+`Pending` state.
+
+```
+$ kubectl get pvc
+NAME             STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+azure-file-pvc   Pending                                      standard       32s
+```
+
+If that is the case, the `persistent-volume-binder` service account does not
+have the relevant Kubernetes RBAC permissions. The storage account creates a
+Kubernetes secret to store the Azure Files Storage Account Key.
+
+```
+Warning    ProvisioningFailed  7s (x3 over 37s)  persistentvolume-controller
```

Comment:
Is this a warning message for the user?

Reply:
@traci-morrison Sorry, this is an output showing the error message from Kubectl. Added the Kubectl command in there now.